Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/Uniform_sizing_field.h

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) 2020 GeometryFactory (France)
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org)
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+//
+//
+// Author(s)     : Jane Tournois
+
+#ifndef CGAL_PMP_REMESHING_UNIFORM_SIZING_FIELD_H
+#define CGAL_PMP_REMESHING_UNIFORM_SIZING_FIELD_H
+
+#include <CGAL/license/Polygon_mesh_processing/meshing_hole_filling.h>
+
+#include <CGAL/Polygon_mesh_processing/Sizing_field_base.h>
+
+#include <CGAL/number_utils.h>
+
+namespace CGAL
+{
+namespace Polygon_mesh_processing
+{
+/*!
+* \ingroup PMP_meshing_grp
+* provides a set of instructions for isotropic remeshing to achieve uniform
```

Comment:
a set of instructions sounds like assembler.

Reply:
Changed to 'provides criteria...' in 1e376316637.